package.json

```diff
@@ -70,9 +70,19 @@
 	},
 	"devDependencies": {
 		"@sindresorhus/tsconfig": "^0.3.0",
+		"@types/array-uniq": "^1.0.1",
+		"@types/cookie": "^0.3.2",
+		"@types/get-res": "^3.0.1",
 		"@types/image-size": "^0.7.0",
+		"@types/lodash.template": "^4.4.5",
+		"@types/log-symbols": "^2.0.0",
 		"@types/node": "^11.11.4",
+		"@types/path-exists": "^3.0.0",
+		"@types/pify": "^3.0.2",
+		"@types/plur": "^3.0.0",
 		"@types/png.js": "^0.2.0",
+		"@types/sinon": "^7.0.6",
+		"@types/viewport-list": "^5.1.1",
```

Comment:
Go through the packages here and check if some have gotten the TS types bundled now.